godotproject/Player.gd

```diff
@@ -191,32 +337,31 @@ func get_tongue_direction():
 	return tdir.normalized()
 
 func _physics_process(delta):
-	update()
```

Comment:
This call to update() was only necessary for the _draw routine; since the _draw was removed, this can be too.